app/controllers/admin/bulk_zombie_url_upload_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class Admin::BulkZombieUrlUploadController < Admin::BulkUploadController
+  def upload
+    handle_bulk_upload(
+      params_key: :bulk_upload_zombie_urls,
+      validator_class: BulkZombieUrls::FileValidator,
+      error_class: BulkZombieUrlUploader::Error,
+      success_path: admin_bulk_zombie_url_upload_index_path,
+      logger_message: 'Zombie Url upload failed'
+    )
+  end
+
+  private
+
+  def set_page_title
+    @page_title = 'Bulk Zombie Url Upload'
+  end
+
+  def success_message(filename)
+    <<~SUCCESS_MESSAGE
+      Successfully uploaded #{filename} for processing.
+      The results will be emailed to you.
+    SUCCESS_MESSAGE
+  end
+
+  def enqueue_job
+    BulkZombieUrlUploaderJob.perform_later(
+      current_user,
+      @file.original_filename,
+      @file.tempfile.path
```

Comment:
This approach won't work on staging or production. When the request comes in the tempfile gets created in the `app` servers but the workers will try to perform the upload from the `crawl` server. 

This file needs to be send to S3 and the job needs to read it from there.

Reply:
Existing similar functionality at https://github.com/GSA/search-gov/blob/main/app/controllers/admin/bulk_url_upload_controller.rb#L33 working fine now in production